Backend/index.js

```diff
@@ -523,6 +531,41 @@ app.get("/notifications", verifyToken, async (req, res) => {
   }
 });
 
+app.post("/notifications/mark-as-read", verifyToken, async (req, res) => {
```

Comment:
i would pull this out into a separate commit next time! so one for addressing previous comments (adding new enum, removing console.logs) and then another one for mark as read

Reply:
alrightt